### ISSUE TITLE
FocusTrapZone and Callout - Attempted import error: 'requireNativeComponent' is not exported from 'react-native'

### ISSUE THREAD

**Issue by frmpiccolo** [microsoft/fluentui-react-native#228]
![fluentui-react-native_issue_ev01](https://user-images.githubusercontent.com/12400108/82711924-267f7400-9c5d-11ea-8534-101d6c8f6165.png)
![fluentui-react-native_issue_ev02](https://user-images.githubusercontent.com/12400108/82711932-28493780-9c5d-11ea-99ac-8c18c30a29d9.png)

I´m trying to include fluentui/react-native into brand new react-native project but it´s not working. 
Message: "Attempted import error: 'requireNativeComponent' is not exported from 'react-native'" (from @fluentui-react-native/focus-trap-zone/lib/FocusTrapZone.js and @fluentui-react-native/callout/lib/Callout.js). 

Environment information: 
- Windows 10 running WSL (Ubuntu 18.04) 
- Node v12.16.2
- react ~16.9.0
- react-dom ~16.9.0
- react-native ~0.61.5 
- react-native-web ~0.11.7
- expo ~37.0.3
- @fluentui/react-native ^0.15.33

Steps to reproduce: 
1. create-react-native-app (Default new app)
2. cd <appname_folder>
3. npm i @fluentui/react-native 
4. Edit App.js
5. Add "import { Checkbox } from '@fluentui/react-native';" into App.js and save it
6. yarn web

**Comment by JasonVMo** [comment]
Good catch and good issue here. Overall we are still rounding out the various platforms and win32 is getting the most attention. That being said we did recently start standing up react-native-web and the recent PR should hopefully address this (along with some other scenarios).

Also we should start paying attention to github issues more in the future, currently more of our tracking is internal but that is changing over time.

**Comment by frmpiccolo** [comment]
> Good catch and good issue here. Overall we are still rounding out the various platforms and win32 is getting the most attention. That being said we did recently start standing up react-native-web and the recent PR should hopefully address this (along with some other scenarios).
> 
> Also we should start paying attention to github issues more in the future, currently more of our tracking is internal but that is changing over time.

Hi Jason, 
Thanks! 
We'll try again after this PR. 
Good job over there and be all safe!

**Comment by JasonVMo** [comment]
I'm going to reopen this as I may be withdrawing or reworking the PR and don't want to lose sight of this. The current PR both delay loads and creates a common cache to address versioning issues.

The delay loading I will likely switch to a more fundamentally delay loaded library (similar to how react-native publishes its components).  We may still want a common cache but that isn't the part that addresses this issue.
